order/models.py

```diff
@@ -8,4 +15,65 @@ class Status(models.Model):
     order = models.PositiveSmallIntegerField(default=0)
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
+
+
+class Declaration(models.Model):
+
+    tracking_code = models.CharField(max_length=13, unique=True, blank=True, null=True)
+
+    cost = models.DecimalField(max_digits=5, decimal_places=2, null=True, blank=True,)
+    discounted_cost = models.DecimalField(max_digits=5, decimal_places=2, null=True, blank=True,)
+    cost_azn = models.DecimalField(max_digits=5, decimal_places=2, null=True, blank=True,)
+    discounted_cost_azn = models.DecimalField(max_digits=5, decimal_places=2, null=True, blank=True,)
+    currency = models.ForeignKey(Currency, on_delete=models.SET_NULL, null=True, blank=True,)
+
+    product_price = models.DecimalField(max_digits=7, decimal_places=2)
+    product_currency = models.ForeignKey(Currency, on_delete=models.SET_NULL, related_name='declarations', null=True, blank=True,)
+    country = models.ForeignKey(Country, on_delete=models.CASCADE, related_name='declarations')
+    product_type = models.ForeignKey(ProductType, on_delete=models.SET_DEFAULT, default=1, related_name='declarations')
+    is_liquid = models.BooleanField(default=False)
+    shop_name = models.CharField(max_length=64)
+    image = models.ImageField(null=True, blank=True, upload_to='images/declaration/')
+    quantity = models.PositiveSmallIntegerField(default=1)
+
+    status = models.ForeignKey(Status, on_delete=models.SET_NULL, null=True, blank=True, related_name='declarations')
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='declarations', blank=True, null=True)
+    discount = models.ManyToManyField(Discount, related_name='declarations', blank=True)
+    weight = models.DecimalField(max_digits=5, decimal_places=3, null=True, blank=True)
+    is_paid = models.BooleanField(default=False)
+    penalty_status = models.BooleanField(default=False)
+    penalty = models.DecimalField(max_digits=4, decimal_places=2, default=0, null=True, blank=True)
+
+    
+    def save(self, *args, **kwargs):
+        if self.cost:
+            azn_rate = float(self.cost)/float(Currency.objects.filter(name="AZN").values_list('rate', flat=True).first())
+            usd_rate = float(self.cost)/float(Currency.objects.filter(name="USD").values_list('rate', flat=True).first())
+
+        if not self.status:
+            self.status = Status.objects.filter(order = 0).first()
+
+        if self.weight and not self.cost:
+            price =  Tariff.objects.filter(Q(max_weight__gte=self.weight) 
+                & Q(min_weight__lte=self.weight) 
+                & Q(country=self.country)).values_list('base_price', flat=True).first()
```

Comment:
Consider having no tariff and having two tariffs for given weight. You should handle them accordingly